dlt/destinations/impl/destination/configuration.py

```diff
@@ -27,6 +28,14 @@ class CustomDestinationClientConfiguration(DestinationClientConfiguration):
     skip_dlt_columns_and_tables: bool = True
     max_table_nesting: Optional[int] = 0
 
+    def fingerprint(self) -> str:
+        """Returns fingerprint of the custom destination.
+
+        Returns:
+            str: The custom destination fingerprint.
+        """
+        return digest128(self.destination_callable.__name__)
```

Comment:
I guess fingerprint for custom destinations is also needed. Otherwise, it's always `''`